### DISCUSSION TITLE
Can't Save Effects

### DISCUSSION THREAD

**Discussion by pkfrulz** [GeekFunkLabs/fluidpatcher#94] [General]
im getting an error when I save the effects . plz help 
see the links I attached ! 

https://github.com/GeekFunkLabs/fluidpatcher/assets/46740764/4556de88-aa86-4316-9d55-bffb733cc087

![WhatsApp Image 2023-11-22 at 4 24 36 PM](https://github.com/GeekFunkLabs/fluidpatcher/assets/46740764/ccbf1e45-33d5-4752-b735-df2ce01ed0d4)

![WhatsApp Image 2023-11-22 at 4 24 35 PM](https://github.com/GeekFunkLabs/fluidpatcher/assets/46740764/7ee59a19-31cf-4eec-9444-83384a4adba7)

**Comment by pkfrulz** [comment]
fixed it on squishbox.py. Line 381. there's missing mode type
![Screenshot 2023-11-22 at 17 46 30](https://github.com/GeekFunkLabs/fluidpatcher/assets/46740764/b43c76a7-9032-4180-b264-e8df06ef87d8)

Thank You

**Reply by albedozero** [reply]
Thanks! This is [fixed](https://github.com/GeekFunkLabs/fluidpatcher/commit/01d75927098c985c75aae4edd96cac72f04331b8) in the latest release. Now I'm curious to see pictures of your full build 😍😉

**Reply by pkfrulz** [reply]
Thank you so much. I'm using this on a standalone drum project. I will upload some when I finished 😍
